tests/queries/query_test.py

```diff
@@ -203,3 +203,6 @@ def test_shell_query(bin_prefix, shell_query, standalone_server):
 
     query = "SELECT 'SHOW ORPHANED DATABASES'; SHOW DATABASES;"
     run_client(bin_prefix, tcp_port, query, b'SHOW ORPHANED DATABASES\ndefault\nsystem\n')
+
+    query = 'USE system;'
+    run_client(bin_prefix, tcp_port, query, b'')
```

Comment:
Thank you for an attempt to fix the issue!

This script is not a test, it's a part of our CI infrastructure, which is responsible for running functional tests with pytest framework (`Functional stateless tests (pytest)` in the list of CI checks). If you want to add a functional stateless test, you should create new `.sql` or `.sh` file and `.reference` file with the corresponding name in `tests/queries/0_stateless/` directory. To run functional test locally use `clickhouse-test` script (you also can use pytest to run it, but it's experimental and not recomended). See also https://clickhouse.tech/docs/en/development/tests/ and https://clickhouse.tech/docs/en/development/continuous-integration/

Moreover, `run_client(...)` runs `clickhouse-client`, which connects to `clickhouse-server` through ClickHouse native protocol. You should use `mysql` client to connect to `clickhouse-server` to test MySQL native protocol. Unfortunately, functional test is not suitable for this. It can be done in integration test (see `tests/integration/test_mysql_protocol/test.py`). Another (probably better) option is to add a unit test (see `src/IO/tests/gtest_mysql_binlog_event_read_buffer.cpp` for example). 

And, of course, it's better to add tests together with code changes in a single PR. I will close this PR, please add new commits to the previous one.

Feel free to ask any questions.